tests/integration/operator/testdata/porter-test-me/Dockerfile.tmpl

```diff
@@ -0,0 +1,21 @@
+FROM debian:stretch-slim
```

Comment:
Heads up that this is an older template. A while back I updated the template to use docker buildkit to speed up builds.